### ISSUE TITLE
Fragpipe Bug

### ISSUE THREAD

**Issue by jjjjjj17** [Nesvilab/philosopher#502]
Hello!! I have a problem running my data, can you help me find out why?
Here is the log file
[log_2024-07-29_23-45-15.txt](https://github.com/user-attachments/files/16423342/log_2024-07-29_23-45-15.txt)

**Comment by liorlobel** [comment]
Dear Fengchao, 

I also get this error with my data. It is a DIA-speclib workflow run.

`time="16:20:04" level=info msg="1+ Charge profile" decoy=126 target=2918
time="16:20:04" level=info msg="2+ Charge profile" decoy=841 target=128944
time="16:20:04" level=info msg="3+ Charge profile" decoy=416 target=53296
time="16:20:04" level=info msg="4+ Charge profile" decoy=84 target=7524
time="16:20:04" level=info msg="5+ Charge profile" decoy=34 target=514
time="16:20:04" level=info msg="6+ Charge profile" decoy=0 target=0
time="16:20:05" level=info msg="Database search results" ions=30379 peptides=21709 psms=194697
time="16:20:05" level=info msg="Converged to 0.78 % FDR with 193196 PSMs" decoy=1501 threshold=0.700198 total=194697
time="16:20:05" level=info msg="Converged to 0.99 % FDR with 17895 Peptides" decoy=178 threshold=0.995283 total=18073
time="16:20:05" level=info msg="Converged to 1.00 % FDR with 26821 Ions" decoy=268 threshold=0.984225 total=27089
time="16:20:05" level=info msg="Protein inference results" decoy=925 target=2409
time="16:20:05" level=error msg="the protein FDR filter didn't reach the desired threshold, try a higher threshold using the --prot parameter"
panic: the protein FDR filter didn't reach the desired threshold, try a higher threshold using the --prot parameter`

Could it be because some of my samples (25 overall, 24 DIA, and 1 DDA for spec library generation) have low identifications? But most samples have decent identifications. I'm not expecting too many because these are immunoprecipitation samples.

Any ideas?

I attach the filter.log file. 

Thanks, 

Lior

[filter.log](https://github.com/user-attachments/files/17189040/filter.log)

**Comment by liorlobel** [comment]
I can provide mzML files, if needed.

**Comment by AimeeD90** [comment]
yes please, and it would be more helpful if you can directly provide the .pep.xml files with this error.
